its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -953,24 +953,17 @@ public void incrementalPrAnalysis_NoCache() throws IOException {
   }
 
   @Test
-  @Ignore
   public void incrementalPrAnalysis_ProducesUnchangedFiles() throws IOException {
-    Assume.assumeTrue(ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(9, 4)); // Cache API was introduced in 9.4
+    Assume.assumeTrue(ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(9, 9)); // Public cache API was introduced in 9.9
 
-    String projectKey = "incremental-pr-analysis";
+    String projectKey = "IncrementalPRAnalysis";
     String baseBranch = TestUtils.getDefaultBranchName(ORCHESTRATOR);
     Path projectDir = TestUtils.projectDir(temp, "IncrementalPRAnalysis");
-    // This is a temporary solution for uploading a valid cache.
-    // The file was generated from https://github.com/SonarSource/sonar-dotnet/commit/28c1224aca62968fb52b0332d6f6b7ef3da11f2b commit.
-    // In order to regenerate this file:
-    // - Update `FileStatusCacheSensor` to trim the paths correctly and save only the relative paths.
-    // - build the plugin
-    // - add it to your SonarQube
-    // - start SonarQube and analyze `IncrementalPRAnalysis` project with `/d:sonar.scanner.keepReport=true`
-    // - archive the content of `.sonarqube\out\.sonar\scanner-report\` (only the content, without parent folder) as `scanner-report.zip`
-    File reportZip = projectDir.resolve("scanner-report.zip").toFile();
```

Comment:
The previous strategy was to upload a full analysis result that included a cache. When this was implemented on the scanner, the analyzers did not have the cache feature implemented so we missed a component that could populate the cache.

Now that we have the analyzers updated we can just run 2 analyses, the first one is full and it will populate the cache, and the second one is for a PR, and is downloading the cache.